examples/simulator-own-addr/client.mjs

```diff
@@ -0,0 +1,47 @@
+const c = await import('@reach-sh/simulator-client');
```

Comment:
I haven't noticed if the others are like this, but you should use the real import syntax

Reply:
@jeapostrophe what's wrong with this syntax? i prefer this syntax because it also works in the node REPL

Reply:
every simulator example is written like this

Reply:
but okay, i'll change it if it's best practice, one moment